### PR TITLE
CHE-437: create loader while opening a stopped workspace

### PR DESCRIPTION
**Pull request by akurinnoy**
**_3 Upvotes**_ Signed-off-by: Oleksii Kurinnyi okurinnyi@codenvy.com

### CODE REVIEW

**Comment by ashumilova** [comment]
+1

**Comment by olexii4** [comment]
+1

**Comment by benoitf** [comment]
+1

File: dashboard/src/app/ide/ide-loader/ide-loader.controller.js [hunk 1]
```diff
@@ -20,10 +20,11 @@ class IdeLoaderCtrl {
    * Default constructor that is using resource
    * @ngInject for Dependency injection
    */
-  constructor(ideSvc, $rootScope, $location) {
+  constructor(ideSvc, $rootScope, $location, cheBranding) {
     this.ideSvc= ideSvc;
     this.$rootScope = $rootScope;
     this.$location = $location;
+    this.branding = cheBranding;
```
Comment (ashumilova) [inline dashboard/src/app/ide/ide-loader/ide-loader.controller.js]: this.branding is not used

**Pull request merged**